app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -0,0 +1,120 @@
+/* ********* Imports: ********* */
+package com.example.sabonit;
+import android.content.Context;
+import android.telephony.TelephonyManager;
+
+
```

Comment:
Good job, seems like this class will serve us well in future. maybe we will add more attributes, according to our needs, because lior and me thought about some extensions in the infrastructure